express/app.mjs

```diff
@@ -0,0 +1,46 @@
+import { createRequire } from 'module';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const require = createRequire(import.meta.url);
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+const layouts = require('express-ejs-layouts');
+const path = require('path');
+const { navRoutes } = require('@contrast/test-bench-utils');
+const express = require('express');
```

Comment:
Let's use `import` wherever we can. This will break instrumentation for the time being, but that's what we want to be testing in the end anyway.